### ISSUE TITLE
My Account Page is missing form labels for accessibility

### ISSUE THREAD

**Issue by currentcreative** [woocommerce/woocommerce#28063]
On Woocommerce My Account page, you get several of these when testing with the WAVE accessibility plugin:

**Missing form label
A form control does not have a corresponding label.**

I found a Woo blogpost from June or July 2019 linking to this page to submit requests / alert you to issues. It's a year later, seems like a form label would have been added to everything by now since people are getting sued left and right for $10,000 a pop by shakedown lawyers for non-accessible websites. Please fix.

**Comment by claudiosanches** [comment]
Hello @currentcreative, 

Thanks for starting this report, but it's missing several steps to reproduce this issue. Currently I can't reproduce, since there's already labels on the template by default, and sounds like it's something related to the theme your website/store is using, or I'm missing some since there's no steps to consistency reproduce this issue.

You can check our default login template: https://github.com/woocommerce/woocommerce/blob/master/templates/myaccount/form-login.php

See the results from WAVE:

![Screenshot from 2020-10-22 20-56-40](https://user-images.githubusercontent.com/1264099/96941580-281dc400-14a9-11eb-84bc-c22943c766bb.png)

Note that warnings are about possible heading, but are false positives.

If you confirm the bug, please open a new issue and provide us with clear steps to reproduce it.
I am closing this issue for now. If I missed something, please leave a comment and we will take a second look.
Thanks!
